e/Ed

```diff
@@ -0,0 +1,25 @@
+The following tab indented lines will cause a "original Unix text editor" (Ed)
```

Comment:
The comments need removed.

Reply:
This file parallels https://github.com/leachim6/hello-world/blob/main/v/Vi. What to do?